tests/php/Control/SimpleResourceURLGeneratorTest.php

```diff
@@ -64,4 +64,12 @@ public function testModuleResource()
             $generator->urlForResource($module->getResource('client/style.css'))
         );
     }
+
+    public function testURLForResourceFailsGracefully()
+    {
+        /** @var SimpleResourceURLGenerator $generator */
+        $generator = Injector::inst()->get(ResourceURLGenerator::class);
+        $url = @$generator->urlForResource('doesnotexist.jpg');
```

Comment:
You probably want to assert a notice is triggered, not silence it like this.

Reply:
Probably, but that must be in another commit not by me... Did I PR against the correct branch?

Reply:
Ah, ok. You'll need to rebase your work on the right branch